scripts/exclude.lst

```diff
@@ -1,7 +1,20 @@
-scripts/*
```

Comment:
Should this list no longer live here? Could the release scripts that use it just use zip_exclude.lst? A little troubled by the near duplication (easy for them to drift apart), and by this list not being referenced by anything in the repo (makes it seems unused).